perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh

```diff
@@ -10,13 +10,19 @@ sudo dpkg --install gcsfuse_"$GCSFUSE_VERSION"_amd64.deb
 cd "${KOKORO_ARTIFACTS_DIR}/github/gcsfuse/perfmetrics/scripts"
 echo Mounting gcs bucket
 mkdir -p gcs
-GCSFUSE_FLAGS="--implicit-dirs --max-conns-per-host 100 --disable-http2 --stackdriver-export-interval=30s"
+GCSFUSE_FLAGS="--implicit-dirs --max-conns-per-host 100 --disable-http2 --debug_fuse --debug_gcs --log-file log.txt --log-format \"text\" --stackdriver-export-interval=30s"
 BUCKET_NAME=gcs-fuse-dashboard-fio
 MOUNT_POINT=gcs
 # The VM will itself exit if the gcsfuse mount fails.
 gcsfuse $GCSFUSE_FLAGS $BUCKET_NAME $MOUNT_POINT
 chmod +x run_load_test_and_fetch_metrics.sh
 ./run_load_test_and_fetch_metrics.sh
+# Copying gcsfuse logs to bucket
+mv log.txt log-$(date '+%Y-%m-%d').txt
```

Comment:
Create a variable and use it everywhere. even in gcsfuse_flags also.

Reply:
Done